gui/UsernamesAndPasswords.java

```diff
@@ -0,0 +1,110 @@
+package com.dyn.admin.gui;
```

Comment:
Can you post a screen shot of this gui? Ill pull it in anyways but just add it in the comments area for the PR